src/Microsoft.OData.Core/Json/JsonWriter.Async.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//---------------------------------------------------------------------
-// <copyright file="JsonWriterAsync.cs" company="Microsoft">
+// <copyright file="JsonWriter.Async.cs" company="Microsoft">

```

Comment:
Is this naming style allowed by the c# naming guidelines and conventions?

Reply:
@ElizabethOkerio I don't know if this is the naming style allowed (or if there's any). I just found it to be an easy way to express the point that _JsonWriter.cs_ and _JsonWriter.Async.cs_ have `JsonWriter` class in common. I'm open to suggestions on doing it better

Reply:
I'm fine with this naming. .NET has similar naming convention for their partial classes, .e.g: https://github.com/dotnet/runtime/blob/9dc6ea62a4d195ae4559f4609a56933c61889756/src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs